frontend/src/components/activity/content/Storyboard.vue

```diff
@@ -55,25 +42,45 @@
                 :filled="layoutMode" />
             </v-col>
             <v-col cols="1">
-              <div v-if="!layoutMode" class="float-right section-buttons">
-                <v-btn icon small class="float-right"><v-icon>mdi-arrow-up-bold</v-icon></v-btn>
-                <dialog-entity-delete :entity="section">
-                  <template #activator="{ on }">
-                    <v-btn icon
-                           small
-                           color="error"
-                           class="float-right"
-                           v-on="on">
-                      <v-icon>mdi-delete</v-icon>
+              <v-container v-if="!layoutMode" ma-0 pa-0>
```

Comment:
```suggestion
              <v-container v-if="!layoutMode" class="ma-0 pa-0">
```
Maybe like this?

Reply:
Changed, but for any reason already works with initial code